tools/metrics/grafana/grafana.go

```diff
@@ -0,0 +1,202 @@
+// This tool is used to view and edit Grafana dashboards locally.
+// See tools/metrics/README.md for more info.
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"os/signal"
+	"path"
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"golang.org/x/sync/errgroup"
+)
+
+var (
+	kube       = flag.Bool("kube", false, "Use kubectl port-forward to point Grafana at real data.")
+	namespace  = flag.String("namespace", "monitor-dev", "k8s namespace")
+	deployment = flag.String("deployment", "prometheus-global-server", "Prometheus server resource to port-forward to.")
```

Comment:
maybe add a comment that these two flags only take effect when kube is true.

Reply:
done